ai/github_issue_handling/issue_prompt_process.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+
+import os
+import json
+
+if __name__ == "__main__":
+    json_data = []
+    extra_labels = []
+    labels_mapping = {'kind/bug': 'bug',
```

Comment:
We can separate these labels based on their types, and add a parameter in script for user to decide how many features they want to involve in the dataset instead of hardcode this label map.

Reply:
Okay, I will make corrections to it later.